src/gui/search/searchsortmodel.cpp

```diff
@@ -47,8 +119,47 @@ bool SearchSortModel::lessThan(const QModelIndex &left, const QModelIndex &right
         if (Utils::String::naturalSort(vL.toString(), vR.toString(), res))
             return res;
 
-        return QSortFilterProxyModel::lessThan(left, right);
+        return base::lessThan(left, right);
+    }
+
+    return base::lessThan(left, right);
+}
+
+bool SearchSortModel::filterAcceptsRow(int source_row, const QModelIndex &source_parent) const
```

Comment:
I mean `QSortFilterProxyModel::filterAcceptsRow()` really has `source_row` and `source_parent` param names, but these names are not typical for Qt (Qt uses camelCase).
And most importantly, I want to say that you can change the names of the parameters in an absolutely free and safely.